src/PowerShell/Microsoft.WinGet.Client.Engine/Commands/Common/BaseCommand.cs

```diff
@@ -37,6 +37,11 @@ internal BaseCommand(PSCmdlet psCmdlet)
                 throw new GroupPolicyException(Policy.WinGet, GroupPolicyFailureType.BlockedByPolicy);
             }
 
+            if (!groupPolicy.IsEnabled(Policy.Configuration))

```

Comment:
With this check we are going to block every winget cmdlet if Configuration policy is disabled because this is shared code for all the winget cmdlets.

I assume we only want to block configuration cmdlet when this policy is disabled?

Reply:
Correct, I was overzealous in my application following everywhere the CLI GP was used.